server/src/__tests__/embedded-languages.test.ts

```diff
@@ -11,7 +11,7 @@ import { generateParser } from '../tree-sitter/parser'
 import { TextDocument } from 'vscode-languageserver-textdocument'
 import { type EmbeddedLanguageType } from '../lib/src/types/embedded-languages'
 import { imports } from '../embedded-languages/python-support'
-import { shebang } from '../embedded-languages/bash-support'
+import { shebang, shellcheckDisables } from '../embedded-languages/bash-support'
```

Comment:
as we add more header lines, we should export them all in one function/attribute.
`embedded_bash_header` or something
```typescript
export const embedded_bash_header = `${shebang}${shellcheckDisables}`
```

Reply:
that way you'll also not need to update this file any more when changing the header.

Reply:
Done.